src/libraries/System.ComponentModel.Composition/src/Microsoft/Internal/GenerationServices.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Internal
     internal static class GenerationServices
     {
         // Type.GetTypeFromHandle
-        private static readonly MethodInfo _typeGetTypeFromHandleMethod = typeof(Type).GetMethod("GetTypeFromHandle");
+        private static readonly MethodInfo _typeGetTypeFromHandleMethod = typeof(Type).GetMethod("GetTypeFromHandle")!;
```

Comment:
Not necessarily for this PR, but at some point it'd be nice to fix the naming here, e.g. s_typeGetTypeFromHandleMethod

Reply:
I can do that here